bplan/management/commands/load_bplan.py

```
@@ -19,209 +19,271 @@
+                e = Point(quadrangle.centroid.x, quadrangle.centroid.y)
+                if e.within(multipolygon[0]):
+                    return e
                 else:
-                    pass
+                    k += 1
             except:
-                pass
-
-            # calculate the Ortsteile in which the bplan is located
-            try:
-                ortsteile = Ortsteil.objects.filter(polygon__intersects=geometry)
-            except:
-                pass
-
-            # Verantwortlichkeiten
-            afs_behoer = feature.get("afs_behoer")
-
-            # Im Verfahren:
-            try:
-                afs_beschl = dateutil.parser.parse(feature.get("afs_beschl"))
-            except:
-                afs_beschl = None
-            try:
-                afs_l_aend = dateutil.parser.parse(feature.get("afs_l_aend"))
-            except:
-                afs_l_aend = None
+                k += 1
 
-            # festgesetzte Verfahren
-            try:
-                f = feature.get("FESTSG")
-                festg = True if f == 'ja' else False
-            except:
-                festg = None
+        return Point(multipolygon[0][0][0])
 
-            try:
-                festsg_von = feature.get("festsg_von")
-            except Exception as e:
-                festsg_von = None
+    def _download_geodata(self, filename, url, layer):
+        call = 'ogr2ogr -s_srs EPSG:25833'\
+            ' -t_srs WGS84 -f'\
+            ' geoJSON %s WFS:"%s%s" %s' % (
+               filename, url, '?TYPENAMES=GML2' if settings.GDAL_LEGACY else '',
+               layer)
+        try:
+            os.remove(filename)
+        except:
+            pass
 
-            try:
-                festsg_am = dateutil.parser.parse(feature.get("festsg_am"))
-            except Exception as e:
-                festsg_am = None
+        try:
+            print('trying to download file from %s' % (url))
+            os.system(call)
```

Can check the return value here? Otherwise there might be silent errors.